queenbee/base/variable.py

```diff
@@ -19,6 +16,7 @@ def get_ref_variable(value: Union[bytes, str]) -> list:
     return parse_double_quotes_vars(value)
 
 
+# TODO: update workflow inputs validation
 def _validate_workflow_var_format(value: str) -> str:
```

Comment:
This is used by the `validate_ref_variable_format` function below but I cannot see any part of the codebase where the `validate_ref_variable_format` function is used. Do we need it for queenbee-luigi? If not I think we can dump these functions entirely as I don't use them in queenbee-argo.

For reference: I think we only use the `{{inputs.some-key}}` form in function commands.

Reply:
I don't use it too. Let's remove it.